proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -1,21 +1,85 @@
 {
   "ams": {
     "color": {
-      "primary-black": { "value": "#000000" },
-      "primary-white": { "value": "#FFFFFF" },
-      "primary-blue": { "value": "#004699" },
-      "primary-red": { "value": "#EC0000" },
-      "dark-blue": { "value": "#102E62" },
-      "orange": { "value": "#FF9100" },
-      "yellow": { "value": "#FFE600" },
-      "green": { "value": "#BED200" },
-      "dark-green": { "value": "#00A03C" },
-      "blue": { "value": "#009DE6" },
-      "purple": { "value": "#A00078" },
-      "magenta": { "value": "#E50082" },
-      "neutral-grey1": { "value": "#E8E8E8" },
-      "neutral-grey2": { "value": "#BEBEBE" },
-      "neutral-grey3": { "value": "#767676" }
+      "border": {
+        "default": { "value": "#181818" }
```

Comment:
No hover variant?

Reply:
Not here; hover belongs in ‘interaction’.

Reply:
Ok so the hover border color will always be the same as the text hover color?